csharp/sources/Valkey.Glide/Internals/Message.cs

```diff
@@ -91,4 +91,4 @@ public void OnCompleted(Action continuation)
     public bool IsCompleted => _completionState == COMPLETION_STAGE_CONTINUATION_EXECUTED;
 
     public object? GetResult() => _exception is null ? _result : throw _exception;
-}
+}
```

Comment:
Please keep a blank line at EOF

Reply:
Artifact of bulk renaming, will fix in a few hours